deployment/src/main/java/com/datastax/oss/quarkus/deployment/internal/CassandraClientProcessor.java

```diff
@@ -64,73 +64,90 @@ FeatureBuildItem feature() {
 
   @BuildStep
   List<ReflectiveClassBuildItem> registerGraphForReflection() {
+    ReflectiveClassBuildItem.Builder builder =
+        ReflectiveClassBuildItem.builder(
+                "org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal")
+            .constructors(true)
+            .methods(true)
+            .fields(true);
     return Arrays.asList(
         // Required for the driver DependencyCheck mechanism
-        new ReflectiveClassBuildItem(
-            true, true, "org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal"),
+        builder.build(),
         // Should be initialized at build time:
-        new ReflectiveClassBuildItem(
-            true, true, "org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerIoRegistryV3d0"),
-        new ReflectiveClassBuildItem(
-            true, true, "org.apache.tinkerpop.shaded.jackson.databind.deser.std.StdDeserializer"),
+        builder
+            .className(
+                new String[] {
+                  "org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerIoRegistryV3d0"
```

Comment:
Can we pass more than one class name here? In this case we might be able to create one builder for all the class names to create items for? (I haven't actually tried it – would need to check the javadocs to understand why this method takes a `String[]` – which is surprising.)

Reply:
Yeah, after reviewing this I realize I have this entirely wrong.  Somehow I misread the [underlying source](https://github.com/quarkusio/quarkus/blob/main/core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/ReflectiveClassBuildItem.java) to say that a single class name could be passed via `ReflectiveClassBuilderItem.builder()` but that String arrays were required for the builder methods.  Basically I missed the vararg in the static builder() method, and what's worse I remember thinking that that mismatch made for a very weird API (_and_ I remembering why the Quarkus guys would do such a thing).. but apparently I never double-checked that.

Anyway, I'll re-work all of this to be more sane... most of what I have here is outright wrong.